owner/manager.go

```diff
@@ -149,7 +155,10 @@ func NewSession(ctx goctx.Context, logPrefix string, etcdCli *clientv3.Client, r
 		if failedCnt%logIntervalCnt == 0 {
 			log.Warnf("%s failed to new session, err %v", logPrefix, err)
 		}
+
+		metrics.NewSessionHistogram.WithLabelValues(logPrefix, metrics.RetLabel(err)).Observe(time.Since(startTime).Seconds())
```

Comment:
We can move this line before error check, move `startTime` before `NewSession` and remove `defer` function.